kong/utils_test.go

```diff
@@ -999,3 +1000,241 @@ func TestFillConsumerGroupPluginDefaults(T *testing.T) {
 		})
 	}
 }
+
+func Test_fillConfigRecord(T *testing.T) {
```

Comment:
```suggestion
func Test_fillConfigRecord(t *testing.T) {
```

to prevent confusion between and accidentally failing the whole test where only a subtest should fail.